include/SFML/Network/Http.hpp

```diff
@@ -338,7 +338,7 @@ class SFML_NETWORK_API Http
     /// \brief Default constructor
     ///
     ////////////////////////////////////////////////////////////
-    Http();
+    Http() = default;
```

Comment:
When do we `= default` in the header and when in the source file?

Reply:
We put it in the header as much as possible unless we're using opaque pointers in which case we're forced to put it in the .cpp file.